_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -11037,12 +11038,29 @@
 	},
 /area/storage/primary)
 "aCx" = (
-/obj/machinery/vending/tool,
 /obj/structure/sign/poster/official/obey{
 	pixel_y = 32
 	},
+/obj/machinery/disposal/deliveryChute{
+	name = "Crate Disposal Chute"
+	},
+/obj/structure/window/reinforced{
+	dir = 4
+	},
+/obj/structure/window/reinforced{
+	dir = 8
+	},
+/obj/machinery/door/window/westleft{
+	base_state = "right";
+	dir = 2;
+	icon_state = "right";
```

Comment:
westleft with all the vars overriden to southright?

Reply:
I'm gonna hit this in another varcleanup soon, however if you can switch this to the proper type for now and remove the rest of the vars, that'd be appreciated.

Reply:
Odd, how did that even happen? I'll fix it.

Reply:
Dirty edits :P